### DISCUSSION TITLE
Empty input return zero

### DISCUSSION THREAD

**Discussion by benzispawn** [uNmAnNeR/imaskjs#600] [Q&A]
**Describe the bug**
When I clear the input with a Number mask I got zero instead of null or '' .

**Environment:**
 - OS: Windows
 - Browser chrome
 - IMask 6.0.7
 - Angular

I`m using in a form to insert data into a database and this "zero" It`s a problem  in my context. Is there any work around??

**Comment by uNmAnNeR** [comment]
if you use `unmask: 'typed'` and you get zero then it's fine. In other cases it should return ''

**Reply by RyKilleen** [reply]
Hey! It's unclear how to use `unmask` with `useIMask`, is there any guidance there?

**Reply by edwinpgm** [reply]
Hi @uNmAnNeR I changed the `unmask` value, but in all moment I am receiving a zero. Do you know how I could solve that?
